.gitignore

```diff
@@ -1,9 +1,9 @@
 *.log
 *.swp
-node_modules/
-public/
-coverage/
+node_modules
+public
+coverage
 **/.DS_Store
-.publish/
+.publish
```

Comment:
IMHO prefer with slash at end, because it is easier to identify that it is a directory,